cmd/native-relayer/cli/start.go

```diff
@@ -75,10 +76,10 @@ var startCmd = &cobra.Command{
 		signal.Notify(c, os.Interrupt, syscall.SIGTERM)
 
 		<-c
-		log.Info().Msg("Stopping the relayer...")
+		log.Info().Msg("\x1b[36mStopping the relayer...\x1b[0m")
```

Comment:
Can you define \x1b by variable and create a function colorized string? Verify simple function or just concat string.

Reply:
Removed the colors as per Robert suggestion